app/internal/export.py

```diff
@@ -12,58 +11,121 @@
 from app.routers.event import get_attendees_email
 
 
-def generate_id(event: Event) -> bytes:
-    """Creates an unique id."""
-    return (
-            str(event.id)
-            + event.start.strftime('%Y%m%d')
-            + event.end.strftime('%Y%m%d')
-            + f'@{DOMAIN}'
-    ).encode()
+def get_icalendar(event: Event, emails: List[str]) -> bytes:
+    """Returns an iCalendar event in bytes.
+
+    Builds an iCalendar event with information from the Event object.
+    and a list of emails.
+
+    Args:
+        event: The Event.
+        emails: A list of emails.
+
+    Returns:
+        An iCalendar that can be used as a string for a file.
+
```

Comment:
Avoid leaving blank line (it is excellent that `"""` is in a separate line, but the extra blank line is redundant)

Reply:
Done.